### DISCUSSION TITLE
Include elements of request in mocked response

### DISCUSSION THREAD

**Discussion by kellynichols** [mitmproxy/mitmproxy#5138] [Q&A] [answered]
Is there any way, using existing features, to mock a response body and include an element of the request in the response? For example, take a UUID from the request URL path and include it in the response body? Perhaps using both `map_local` and `modify_body`? Or would this best be achieved as an add on? Thanks!

**Comment by mhils** [comment, marked as answer]
I think this would be best achieved in a custom addon. :-)

**Reply by kellynichols** [reply]
Thank you!
